tests/test_commands.py

```diff
@@ -126,3 +135,41 @@ def test_output_listing(mapdl, plastic_solve, func, args):
     if HAS_PANDAS:
         out_df = out.to_dataframe()
         assert isinstance(out_df, pd.DataFrame) and not out_df.empty
+
```

Comment:
Two lines between functions.
```suggestion


```

Reply:
I saw you had seen this quite a lot in the previous PR. I need to get my head around that.